### PR TITLE
Add Blob storing AWS S3 Integration

### PR DESCRIPTION
**Pull request by liangshiw**
Resolve https://github.com/abpframework/abp/issues/4610

### CODE REVIEW

**Comment by maliming** [comment]
Please add it to the `common.ps1`

https://github.com/abpframework/abp/blob/dev/nupkg/common.ps1#L62

File: framework/src/Volo.Abp.BlobStoring.Aws/Volo/Abp/BlobStoring/Aws/AwsTemporaryCredentialsCacheItem.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Volo.Abp.BlobStoring.Aws
+{
+    [Serializable]
+    public class AwsTemporaryCredentialsCacheItem
+    {
+        public const string Key = "AwsBlobTemporaryCredentialsCache";
```
Comment (maliming) [inline framework/src/Volo.Abp.BlobStoring.Aws/Volo/Abp/BlobStoring/Aws/AwsTemporaryCredentialsCacheItem.cs:8]: Use calculated key instead of fixed one.

**Comment by hikalkan** [comment]
That's great. Did we document this?

**Comment by realLiangshiwei** [comment]
Yes, https://github.com/abpframework/abp/pull/4842

**Pull request merged**